sphinx/source/quickstart/oeengine.rst

```diff
@@ -0,0 +1,55 @@
+OE Engine Walkthrough
+=====================
+
+:term:`OpenEnclave Engine` (OE Engine) offers detailed `deployment instructions`_, but this is a very condensed summary to get a CCF-ready VM up and running in 5 minutes. You can either execute these steps on a machine with the `Azure CLI`_ installed, or use `Azure Cloud Shell`_.
+
```

Comment:
maybe add a note here saying that the pre_vm_make.sh script will install the Azure CLI and the oe-engine binary?